syslog/receiver.go

```diff
@@ -11,9 +11,17 @@ import (
 )
 
 var (
-	syslogAddr = conf.SyslogBind
-
-	loggerFuncs map[byte]func(...interface{})
+	syslogAddr  = conf.SyslogBind
+	loggerFuncs = map[byte]func(...interface{}){
```

Comment:
this appears to only be used at line 122. maybe don't need a package variable https://github.com/tinkerbell/boots/blob/e58c69257db494a216b41813b27f0d230dd195c6/syslog/receiver.go#L122

Reply:
very true, will drop